### ISSUE TITLE
Evince 실행 안됨

### ISSUE THREAD

**Issue by parkmino** [dasom-im/dasom#53]
안녕하세요, 덕분에 우분투 15.10 32비트에서 다솜을 잘 쓰고 있습니다.

언제부터인지는 모르지만, evince를 실행하면 아래와 같은 에러메시지가 나오면서, 화면창에 아무것도 안보이네요.
비슷한 어플인 qpdfview는 문제가 없습니다.
(첫 줄은 원래 다른 명령어에서도 나오는 메시지라 무시하셔도 됩니다)

*\* (evince:5101): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-QJzxda75FE: Connection refused

(evince:5101): Gtk-CRITICAL **: gtk_widget_hide: assertion 'GTK_IS_WIDGET (widget)' failed

(evince:5101): Gtk-CRITICAL **: gtk_widget_hide: assertion 'GTK_IS_WIDGET (widget)' failed

(evince:5101): dasom-CRITICAL **: dasom-im.c:495: dasom_im_init: Could not connect: Permission denied

(evince:5101): dasom-CRITICAL **: dasom-im.c:495: dasom_im_init: Could not connect: Permission denied
...

**Comment by ghost** [comment]
입력기 문제는 아닌 것 같네요. evince 프로젝트에 버그 리포트 해주세요.
아마, root 권한으로 뭔가를 하셨을 것 같은데... 잘 생각해보세요.

**Comment by parkmino** [comment]
새로 설치해도 마찬가지네요.
다른 분들한테서 문제가 없다면 이 이슈는 닫는게 맞겠습니다.

수고하세요~

**Comment by ghost** [comment]
`apparmor` 때문에 evince 에서 접속 실패합니다.

**Comment by ghost** [comment]
@parkmino 

root 권한으로 `/etc/apparmor.d/abstractions/dasom` 파일을 만들어 주시고 아래의 내용을 넣어주세요.

```
# vim:syntax=apparmor

  # Allow connecting to the dasom-daemon socket
  unix (send, receive, connect)
       type=stream
       peer=(addr="@unix:abstract=dasom"),
```

root 권한으로 `/etc/apparmor.d/usr.bin.evince` 파일을 연 후
`#include <abstractions/ibus>` 다음 줄에 `#include <abstractions/dasom>` 를 추가해주세요.

그 후 재부팅하면 됩니다.

**Comment by ghost** [comment]
`/etc/apparmor.d/usr.bin.evince` 파일에 ibus 가 두번 나타납니다.
`#include <abstractions/dasom>` 이 한줄 더 추가 되어야합니다.

**Comment by parkmino** [comment]
해결되었고, 신경 써 주셔서 감사합니다.
오랜만에 Evince를 보니 반갑네요 ^ ^

**Comment by ghost** [comment]
Nimf 는 Dasom 을 계승하는 입력기 프레임워크입니다.
evince debian/apparmor-profile 에 nimf 를 지원해달라고 버그 리포트를 보냈습니다.
https://bugs.launchpad.net/ubuntu/+source/evince/+bug/1569863

링크를 방문하신 후 로그인 하시면, [This bug affects you](https://bugs.launchpad.net/ubuntu/+source/evince/+bug/1569863/+affectsmetoo) 링크가 있습니다.
그걸 클릭하셔서 `Yes, it affects me` 선택해주시면 되겠습니다.
